imagenet1k_alexnet/run_results/A100/a100_4_1024.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+#SBATCH --job-name=alex_a100_4_1024
+#SBATCH --output=logs/alex_a100_4_1024_%j.out
+#SBATCH --error=logs/alex_a100_4_1024_%j.err
+#SBATCH --partition=kempner
+#SBATCH --account=kempner_dev
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=4      # Number of processes (1 per GPU)
+#SBATCH --gres=gpu:4             # Total GPUs per node
+#SBATCH --cpus-per-task=16       # Number of CPU cores per task
+#SBATCH --mem=244GB              # Memory per node
+#SBATCH --time=36:00:00          # Maximum runtime
+#SBATCH --constraint=a100
+
+
+# User and Envrironment Variables ----------------------------------------------
+
+# Starting the changing and customizing the script from here ------
+
+# User Variables ---------------
+ARCHITECTURE=alexnet  # Model architecture (e.g., resnet50)
+EPOCHS=90              # Number of epochs to train
+BATCH_SIZE=1024         # Total batch size 
+LEARNING_RATE=0.1      # Initial learning rate
+WORKERS=16             # Number of CPU workers for data loading
+
+# Environment Variables --------
+# Environment Variables --------
+CONDA_ENV=/n/holylfs06/LABS/kempner_shared/Everyone/common_envs/miniconda3/envs/pytorch_image # <--- Conda environment path
+
+
+# End of the changing and customizing the script ------------------
+# DON'T CHANGE ANYTHING BELOW THIS LINE UNLESS YOU KNOW WHAT YOU ARE DOING
+#-------------------------------------------------------------------------------
+
+
+PYTHON_BINARY=$CONDA_ENV/bin/python
+DATA_DIR="/n/holylfs06/LABS/kempner_shared/Lab/data/imagenet_1k"
+RESNET_SCRIPT_DIR=./main.py
```

Comment:
Please add `PROJECT_DIR` and generate full path for running script.

Reply:
Added
PROJECT_DIR=/n/netscratch/kempner_dev/Lab/bdesinghu/Vision/AlexNet/
Changed RESNET_SCRIPT_DIR to SCRIPT_DIR